go/vt/dbconfigs/dbconfigs.go

```diff
@@ -164,10 +165,10 @@ func (dbcfgs *DBConfigs) String() string {
 
 // Redact will remove the password, so the object can be logged
 func (dbcfgs *DBConfigs) Redact() {
-	dbcfgs.App.ConnectionParams.Redact()
-	dbcfgs.Dba.Redact()
-	dbcfgs.Filtered.Redact()
-	dbcfgs.Repl.Redact()
+	dbcfgs.App.ConnectionParams.Pass = mysql.RedactedPassword
```

Comment:
Redacting passwords seems like a higher-level function than the level of the mysql connection driver. Maybe we should move mysql.RedactedPassword into package dbconfigs? Is mysql.RedactedPassword used anywhere else?